server/views/formPages/sentencePlan.html

```diff
@@ -0,0 +1,27 @@
+{% extends "../partials/layout.html" %}
+
+{% from "breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "button/macro.njk" import govukButton %}
+
+{% block beforeContent %}
+{{ govukBreadcrumbs({
+  items: [
+    {
+      text: "Home",
+      href: "#"
+    },
+    {
+      text: "Offender summary",
+      href: "/offenderSummary/crn/" + crn
```

Comment:
offender-summary?

Reply:
is there no testing in this project :disappointed: ?  Would have thought that :point_up: would have caused a test to fail.  Should really have unit and integration tests...